app/components/Options/Config.js

```diff
@@ -165,6 +165,16 @@ export default {
             '#content .section-wrapper [class^=\'icon-\']'
           ]
         },
+        {
+          type: 'FontFamily',
+          optionFonts: [
+            'None',
+            'Open Sans',
+            'MS Trebuchet',
+            'Helvetica',
+            'Arial'
```

Comment:
Something that'd be nice to offer is the native OS font, which is what Github's font is. That's Segoe UI on Windows and it's San Francisco on Mac / iPhone. The CSS declaration for that looks something like this: 

```
font-family: system,-apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica Neue,Helvetica,Lucida Grande;
```

What do you think about adding that one to this list?